changelog/fragments/01-move-away-from-operator-sdk-binary.yaml

```diff
@@ -0,0 +1,11 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Add a Parameter to the Makefile to allow modifying the
+      'operator-sdk' binary before running any commands.
+      This allows you to run `OPERATOR_SDK=~/local/operator-sdk-1.2 make bundle`
+      without modifying and of the default makefile provided
+    kind: addition
+
+    breaking: false
```

Comment:
@camilamacedo86 do you think we need a migration step for this change? It definitely isn't required to work but not sure. Thoughts?

Reply:
@jmrodri seems this didn't get any ❤️ 
I will unrelated move to rebase this